pycbc/inference/option_utils.py

```diff
@@ -262,13 +262,23 @@ def add_plot_posterior_option_group(parser):
     """
     pgroup = parser.add_argument_group("Options for what plots to create and "
                                        "their formats.")
-    pgroup.add_argument('--plot-marginal', action='store_true', default=False,
+    pgroup.add_argument('--plot-marginal', default=None, type=str,
```

Comment:
You should consider adding a ``choices=["percentiles", "hpd"]`` here.

Reply:
Done.